app/models/ems_event.rb

```diff
@@ -10,14 +10,18 @@ class EmsEvent < EventStream
     'Rename_Task',
   ]
 
+  CLASS_GROUP_LEVELS = %i[critical warning].freeze
```

Comment:
For i18n, I think the better way to do this might be to make this:

```suggestion
  CLASS_GROUP_LEVELS = [
    N_("Critical"),
    N_("Warning")
  ].freeze
```

Then later, or perhaps in another constant, do `CLASS_GROUP_LEVELS.map { |l| l.downcase.to_sym }`

This way, the strings are in the translation catalog, but the code can still use symbols.